specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2019-01-21/kusto.json

```diff
@@ -1460,6 +1817,85 @@
       },
       "description": "Azure capacity definition."
     },
+    "SkuDescriptionList": {
+      "description": "The list of the EngagementFabric SKU descriptions",
```

Comment:
Is this description correct - EngagementFabric?  I haven't seen that phrase in this spec.

Reply:
No, I'll fix it, thanks